### ISSUE TITLE
Correction in the systemd.timer manual....

### ISSUE THREAD

**Issue by MrM40** [systemd/systemd#18998]
In the sytemd-timer manual, section [Table 1. Settings and their starting points](https://www.freedesktop.org/software/systemd/man/systemd.timer.html#OnActiveSec=) you have this these descriptions:

```
OnUnitActiveSec=   | Defines a timer relative to when the unit the timer unit is activating was last activated.
OnUnitInactiveSec= | Defines a timer relative to when the unit the timer unit is activating was last deactivated.
```
Could you consider rephrase, in my view it doesn't make sense.

**Comment by boucman** [comment]
what do you consider unclear ? The sentence is a bit hard to parse, but it's not that bad and precisely explain what's going on...

How would you rephrase it ?
"Will trigger a certain time after the target unit was last activated" maybe ?

**Comment by MrM40** [comment]
I admit I'm not native english but "the unit the timer unit is activating was last deactivated" seems very wrong.

**Comment by MrM40** [comment]
do you mean .
`Defines a timer relative to when the timer unit was last activated.`

**Comment by boucman** [comment]
No. it's relative to the triggered unit, not the timer unit (that one is OnActiveSec=)

**Comment by MrM40** [comment]
But yes "Will trigger a certain time after the target unit was last activated" is sure more readable and understandable :-)

**Comment by poettering** [comment]
timers can activate any kind of unit, not just `.target` units, hence I doubt your proposal is more understandable?

**Comment by poettering** [comment]
i must say i think the current wording is actually OK. Unless a convincing better wording is supplied I think we should close this

**Comment by MrM40** [comment]
It's difficult to suggest a phrase when I have no idea what the current phrase is  saying.
But if two independent people says it's okay I guess it must be. Or are you sharing the same non-english native language?
Maybe that's why it make sense to you?
But in my view `when the unit the timer unit is activating was last activated` is completely rubbish and it not correct englsh.

**Comment by boucman** [comment]
It is a convoluted sentence and the repetition of "unit" and "activating/activated" makes it hard to parse, but it's a valid english sentence and it is a very precise description of what's happening.

Ideally we would find a more clear sentence to express that, but the current one is "englishly correct"

**Comment by poettering** [comment]
I am totally open to better wording, I am not a a native English speaker, but the one suggested so far weren't accurate...

**Comment by poettering** [comment]
let's close this. I am happy to review/merge better wording, but I think the current version is good enough for now, and as I am not a native speaker I doubt I am the best person to come up with a perfect replacement here.

Hence, if you want something don in this area, please submit a PR with a better (but accurate) wording.
